src/Illuminate/Auth/Notifications/ResetPassword.php

```diff
@@ -44,9 +52,12 @@ public function via($notifiable)
      */
     public function toMail($notifiable)
     {
+        $email = $notifiable->getEmailForPasswordReset();
+        $link = url("password/reset?email={$email}&expiration={$this->expiration}&token={$this->token}");
```

Comment:
The URL has always been hard-coded, because Laravel is assuming you're using the `Router::auth()` routes.

If you want to use a custom route, you can use your own notification by adding a `sendPasswordResetNotification()` method to your class. See [CanResetPassword](https://github.com/laravel/framework/blob/master/src/Illuminate/Auth/Passwords/CanResetPassword.php) trait.